### ISSUE TITLE
HTTP response 500 Error

### ISSUE THREAD

**Issue by mzch** [BookStackApp/BookStack#1634]
**Describe the bug**

HTTP response 500

**Steps To Reproduce**
1. Fresh install 0.27.3
2. Access BookStack thru a browser

- Apache Log
```
[Thu Sep 05 00:41:48.015878 2019] [proxy_fcgi:error] [pid 2071:tid 139931225544448] [client 2405:6584:7840:f00:ccdc:7d6c:99c2:317f:55643] AH01071: Got error 'PHP message: PHP Fatal error:  Uncaught ReflectionException: Class view does not exist in /srv/BookStack/vendor/laravel/framework/src/Illuminate/Container/Container.php:752\nStack trace:\n#0 /srv/BookStack/vendor/laravel/framework/src/Illuminate/Container/Container.php(752): ReflectionClass->__construct('view')\n#1 /srv/BookStack/vendor/laravel/framework/src/Illuminate/Container/Container.php(631): Illuminate\\Container\\Container->build('view')\n#2 /srv/BookStack/vendor/laravel/framework/src/Illuminate/Container/Container.php(586): Illuminate\\Container\\Container->resolve('view', Array)\n#3 /srv/BookStack/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(732): Illuminate\\Container\\Container->make('view', Array)\n#4 /srv/BookStack/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php(110): Illuminate\\Foundation\\Application->make('view', Array)\n#5 /srv/BookStack/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php(971): app('Illuminate\\\\Cont...')\n#6 /srv/BookStack/vendor/...'
```

**Expected behavior**
See app screen.

**Screenshots**

**Your Configuration (please complete the following information):**
 - Exact BookStack Version (Found in settings): 0.27.3
 - PHP Version: 7.3
 - Hosting Method (Nginx/Apache/Docker): Apache

**Additional context**

**Comment by ssddanbrown** [comment]
Hi @mzch,
Sorry to hear you are having issues.

* When you ran `composer install` during install, or even now, do you see any errors appear? 
* Have you set the permissions of the `storage/` and `bootstrap/cache/` folders so that Apache can write to them?

**Comment by mzch** [comment]
Sorry for my reply delayed.
This issue is fixed by updating to v0.27.4.
Thanks.
